react-charts/area-chart/step2.md

```diff
@@ -1,24 +1,49 @@
-We'll begin by adding a CSS selector to define the chart size...
-
-- Katacoda is setting up a new React application for you. You'll be ready to code once the server starts and you can see "Welcome to PatternFly" on the lower pane.
-
-## Task
-
-1) Navigate to the `src` folder and open `src/app.css`{{open}}
-
-2) Next we'll add some CSS to define the overall chart size
-
-Copy the following code into the app.css file, replacing all of the content there:
-
-<pre class="file" data-filename="src/app.css" data-target="replace">
-.chart-container {
-  height: 250px;
-  width: 600px;
+Create a simple area chart.
+
+1) <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>
+
+2) <strong>Set up the general structure of the chart and the necessary import statements.</strong>
+
+Copy the following code into the `App.js` file, replacing all of the content there:
+
+<pre class="file" data-filename="App.js" data-target="replace">
+import React from 'react';
+import "@patternfly/react-core/dist/styles/base.css";
+import './app.css';
+import {
+  Chart,
+  ChartAxis,
+  ChartArea,
+  ChartGroup,
+  ChartThemeColor,
+  ChartVoronoiContainer
+} from '@patternfly/react-charts';
+
+function App() {
+  return (
+    &lt;div className=&quot;chart-container&quot;&gt;
+      &lt;Chart
+        height={250}
+        width={600}
+      &gt;
+        &lt;ChartArea
+          data={[
+            { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
+            { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
+            { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
+            { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
+          ]} 
+          interpolation="monotoneX"
+        /&gt;
+      &lt;/Chart&gt;
+    &lt;/div&gt;
+  );
 }
+
+export default App;
 </pre>
 
-- Because Victory renders responsive containers, the Chart's width and height props do not determine the width and height of the chart in number of pixels, but instead define an aspect ratio for the chart. 
-The exact number of pixels will depend on the size of the container the chart is rendered into. 
-Typically, the parent container is set to the same width in order to maintain the aspect ratio.
+The `interpolation` property adds interpolating curves.
```

Comment:
Should we link to PatternFly docs for more details on this? This description doesn't provide much useful information. Seeing the same issue with some of the props described in other modules.

Reply:
@dlabrecq what do you think? is this straight from victory ?

Reply:
You do link to victory in the start, so maybe when there is a new prop introduced we ask the user to link back to victory documentation

Reply:
Even if we copy some more detailed descriptions from the PatternFly or Victory docs, that would be an improvement. It just seems like this is a really vague description as is.

Reply:
My intention was to point out that Victory renders responsive containers. The height and width are not used in the way users typically expect. 

IMO, it's a mistake to remove this information -- please add this back in

Reply:
This info is still in the tutorial. It's in step one.